app/assets/stylesheets/admin.scss

```diff
@@ -29,7 +29,7 @@ body > div.container {
 
 .users {
   li.user {
-    border-bottom: 1px solid $light-grey;
+    border-bottom: 1px solid $text-grey;
```

Comment:
Oh, this is actually the only place where I should have kept `$light-grey` because we import `colors.scss` at the beginning of the file. Will change this back to `$light-grey`. I didn't change the value of `$light-grey` because we also use it in the mobile version.